common/src/dodeca.rs

```diff
@@ -281,6 +286,45 @@ lazy_static! {
 
         result
     };
+
+    /// given a side s and a vertex v, returns a vertex that is adjacent to v but not
+    /// incident on s, if such a vertex exists
+    static ref PERPENDICULAR_VERTEX: [[Option<Vertex>; VERTEX_COUNT]; SIDE_COUNT] = {
+        let mut result = [[None; VERTEX_COUNT]; SIDE_COUNT];
+
+        for side in Side::iter() {
+            let incident_vertices = side.vertices();
+
+            // 'v' and 'vertex as usize' will have different values.
+            #[allow(clippy::needless_range_loop)]
```

Comment:
This line (and the comment above) can now be removed. After applying my suggested changes, Clippy no longer warns you about this code.